src/VlcVideoOutput.h

```diff
@@ -4,10 +4,9 @@
 #include <memory>
 #include <mutex>
 #include <atomic>
-
 #include <uv.h>
-
 #include <libvlc_wrapper/vlc_vmem.h>
+#include <cstring>
```

Comment:
1) since `memset`/`memcpy` does not used in header, it's better move include to .cpp
2) it will be enough just add `#include <string.h>` there. There will be no need to use `std::` then.